### DISCUSSION TITLE
Raising PHPMailer's minimum PHP version

### DISCUSSION THREAD

**Discussion by Synchro** [PHPMailer/PHPMailer#3093] [Ideas]
PHPMailer currently still supports running on PHP back to 5.5. I've been thinking about making a major release with BC breaks that raises the minimum PHP version to 7.4, to match [WordPress' minimum](https://wordpress.org/about/requirements/). This will allow me to remove quite a bit of legacy cruft and ease future maintenance. PHP 7.4 is already long obsolete anyway, and nobody should be doing anything new with it, or any older version of PHP. Thoughts?

**Comment by jrfnl** [comment]
The minimum requirement for WordPress is actually PHP 7.2 (since WP 6.6 = current major), though PHP >= 7.4 is (strongly) recommended.

**Reply by jrfnl** [reply]
To quote the page you linked to:

> Note: If you are in a legacy environment where you only have older PHP or MySQL versions, WordPress also works with PHP 7.2.24+ and MySQL 5.5.5+. However, these versions have reached their official End Of Life and may expose your site to security vulnerabilities.

**Reply by Synchro** [reply]
Well, even 7.2 is a decent step up from 5.5!

**Reply by jrfnl** [reply]
It is indeed. And PHP 7.4 might not be a long wait anyway, so that drop could follow in a year or so (both for WP as well as PHPMailer).

**Comment by campbell-m** [comment]
My vote is for 7.2 rather than 7.4. We use PHPMailer in our application ([MRBS](https://github.com/meeting-room-booking-system)) and our minimum version is 7.2. There are still quite a few sites out there running 7.2, rightly or wrongly.
